dotCMS/src/main/java/com/dotcms/vanityurl/model/VanityUrl.java

```diff
@@ -139,4 +139,4 @@
 
     Map<String, Object> getMap() throws DotDataException, DotSecurityException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)